spec/rspec/core_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe RSpec do
   fake_libs = File.expand_path('../../support/fake_libs', __FILE__)
   allowed_loaded_features = [
+    /open3.rb/,       # Used by Bisect::Runner, which is not normally loaded
```

Comment:
My reading of this is that this would not be loaded except in specs utilising the bisect feature, can we not use the isolated environment stuff to test that feature avoiding this?